src/main/java/org/pac4j/vertx/config/Pac4jConfigurationFactory.java

```diff
@@ -41,15 +45,33 @@
  * @author Jeremy Prime
  * @since 2.0.0
  */
-public class Pac4jConfigurationFactory {
+public class Pac4jConfigurationFactory implements ConfigFactory {
```

Comment:
This is something we didn't agree upon: I'm just proposing to use the ConfigFactory here.... Because this is the kind of component you need: something to create the config. So I prefer a kwon factory than a static method...